std-bits/table/src/main/java/org/enso/table/data/column/operation/map/bool/BooleanIsInOp.java

```diff
@@ -60,43 +60,30 @@ public Storage<?> runZip(
   }
 
   private BoolStorage run(BoolStorage storage, boolean hadNull, boolean hadTrue, boolean hadFalse) {
```

Comment:
I think they all mean the same concept here really, so it is just a bit of chaos that got into the codebase over time.

- `Nothing` is the Enso-centrict name.
- `Null` probably came from the Java internals naming convention (where the `Nothing` is often represented as a `null` reference).
- _missing_ is the naming that stemmed from also old version of Table (we used to have `Column.is_missing` instead of `Column.is_nothing`; the external API got renamed, but the internals likely stayed. The `*Missing` bitsets are used when we are storing values like integers in unboxed `long[]` arrays - such don't take `null` (like `Object[]` could), so there needs to be a separate flag of which values are missing. I guess it could be called `isNothing` though, for consistency.

We could probably rename all of these to `nothing`-based names.